src/collection.js

```diff
@@ -156,6 +156,7 @@ export default class Collection {
       { data, permissions },
       {
         last_modified,
+        patch: !!patch,
```

Comment:
I know that I'm the one who suggested you to put `patch: false` when unspecified, but I see that is not the way it's done elsewhere :/ eg. https://github.com/Kinto/kinto-http.js/blob/master/src/bucket.js#L150